config/routes.rb

```diff
@@ -15,6 +15,8 @@
     get "nonprofits"
   end
 
+  resources :student_applications, only: [:new, :create], path: "apply/students"
```

Comment:
Not super sure what the `path:` does since I've never used it - could you explain / link me? :)